docker-compose.yml

```diff
@@ -1,22 +1,26 @@
-version: '3'
+version: '3.7'
 
 services:
   rdf-differ:
     container_name: rdf_differ
     build: .
-    image: rdf_differ:latest
+    command: flask run --host=0.0.0.0 --port=${RDF_DIFFER_API_PORT}
```

Comment:
let's discuss the difference between guniorn and flask running, even for the dev environment.